README.md

```diff
@@ -1 +1,66 @@
-# Microverse-Tic-Tac-Toe-Ruby
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Tic Tac Toe
+
+In order to win the game, a player must place three of their marks in a horizontal, vertical, or diagonal row.
+Players will soon discover that the best play from both parties leads to a draw. 
+
+## How do we play Tic Tac Toe?
+
+- Run main on your terminal
+- The game is played on a grid that's 3 squares by 3 squares.
+- You are X, secound player is O. Players take turns putting their marks in empty squares.
+- The first player to get 3 of her marks in a row (up, down, across, or diagonally) is the winner.
+- When all 9 squares are full, the game is over.
+
+
+## Screenshot
+
+![Screenshot](tic.png)
+
+## Built With
+ [RUBY](https://github.com/AymanJabr/Microverse-Tic-Tac-Toe-Ruby)
+
+## Getting Started
+
+To get started with ````Tic Tac Toe```` you need to follow the intructions to be able to play it on your local machine.
+
+- To get a local copy up and running follow these simple example steps.
+
+- Clone this repository with git clone ```https://github.com/AymanJabr/Microverse-Tic-Tac-Toe-Ruby.git``` using your terminal or command line.
+- Change to the project directory by entering :
```

Comment:
Include the `git clone` inside the quotes ` `` `, so the full command will be like `git clone https://github.com/AymanJabr/Microverse-Tic-Tac-Toe-Ruby.git`, this is the full command that they need to clone the repo.